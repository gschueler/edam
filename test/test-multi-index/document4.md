% Document4

Document4