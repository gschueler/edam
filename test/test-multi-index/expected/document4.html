<!DOCTYPE html>
<html>
<head>
  <title>Document4</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta name="generator" content="pandoc" />
  <link rel="stylesheet" href="style.css" type="text/css" />
</head>
<body>


<nav class="page top">
    <ul>
        <li class="current"><a href="document4.html">Chapter 4: Document4</a></li>
        <li class="toc"><a href="index.html">Test Multi Index</a></li>
        <li class="prev"><a href="document3.html">Chapter 3: Document3</a></li>
        
    </ul>
</nav>
<div id="docbody">
<h1 class="title">Document4</h1>
<p
>Document4</p
>
</div>

<nav class="page bottom">
    <ul>
        <li class="current"><a href="document4.html">Chapter 4: Document4</a></li>
        <li class="toc"><a href="index.html">Test Multi Index</a></li>
        <li class="prev"><a href="document3.html">Chapter 3: Document3</a></li>
        
    </ul>
</nav>

</body>
</html>